manual/src/main/asciidoc/whats-new.adoc

```diff
@@ -51,26 +51,85 @@ More details about JSON Schemas implementation are available in the <<JSON schem
 
 The introduction of JSON schema required us to modify Apache Unomi data model, One of the key differences is the removal of open maps.
 
-For example, the following event model in Apache Unomi 1.x:
+The properties field in the events objects provided by unomi are now restricted by JSON schema.
+It's not possible to add value in this property without adding the JSON schema which allows to add the properties.
+
+If there is dynamic properties that you want to add to the event, you should use the new property of the event named flattenedProperties.
```

Comment:
You should rephrase a bit this paragraph to introduce `flattenedProperties` before instructing to use it.

```
A new property, flattenedProperties has been introduced to... this property is ...
```